op-node/metrics/metrics.go

```diff
@@ -116,6 +120,9 @@ type Metrics struct {
 
 	TransactionsSequencedTotal prometheus.Counter
 
+	// Store a map of peer to their score so we can build a score histogram
+	PeerStore map[peer.ID]float64
+
```

Comment:
Is this still used? Please make sure to clean up all unused artifacts from the first version of the implementation.